README.md

```diff
@@ -75,6 +75,18 @@ Convert workout data from Peloton into a TCX file that can be uploaded to Garmin
 * A TCX file for each workout will be created in the `output` directory
 * The resulting TCX file can then be uploaded to Garmin
 
+## Python Versions Tested
+* Windows:          3.6.4
+
+* Mac (10.15.5):      3.7.7
+
+* Linux
+    * Ubuntu (20.04): 3.8.2
+    * CentOS (7):     3.6.8
+
+
+Most likely, these python versions will work on any OS. 
```

Comment:
Thanks for doing this research and documenting! I recently (literally last night) setup unit testing and some github actions that make it much easier to automate moving forward as well.  

You could also add a link to [this file](https://github.com/philosowaffle/peloton-to-garmin/blob/master/.github/workflows/pr-test.yml#L18).  Every PR is now validated on those combinations of OS's and Python versions.

Reply:
ha. I noticed that. I didn't think that was there before. Yeah, I'll rework that page at some point now that I know the testing situation. We will see how much the heat takes out of me tomorrow on my ~135k.

Reply:
Good luck!